surreal.js

```diff
@@ -36,37 +36,65 @@ var $ = { // You can use a different name than "$", but you must change the refe
 		e.attributes    = e.attribute
 		e.attr          = e.attribute
 
+		// Tree Traversal.
+		e.el = (selector, start = e, warning = true) => $.el(selector, start, warning)
+		e.any = (selector, start = e, warning = true) => $.any(selector, start, warning)
+		Object.defineProperty(e, 'parent', {
+			get: () => $.sugar(e.parentElement),
+			configurable: true
+		})
+
+
 		// Add all plugin sugar.
 		$._e = e // Plugin access to "e" for chaining.
-		for (const [key, value] of Object.entries(sugars)) {
+		for (const [key, value] of Object.entries($.sugars)) {
```

Comment:
I can't figure out what changed, but at some point this was giving me an error that `sugars` was undefined until I added the `$.`.